centreon-open-tickets/www/modules/centreon-open-tickets/class/rule.php

```diff
@@ -291,15 +305,23 @@ public function loadSelection(?CentreonDB $dbStorage, string $cmd, string $selec
                 SQL;
 
             if (!$centreon_bg->is_admin) {
-                $query .= <<<'SQL'
+                $aclGroupIdsCondition = '';
+                foreach (explode(',', str_replace("'", '', $centreon_bg->grouplistStr)) as $aclId) {
+                    if (empty($aclGroupIdsCondition)) {
+                        $aclGroupIdsCondition .= ':acl_' . $aclId;
+                    } else {
+                        $aclGroupIdsCondition .= ', :acl_' . $aclId;
+                    }
```

Comment:
simpler like this :)
```suggestion
                    if (! empty($aclGroupIdsCondition)) {
                        $aclGroupIdsCondition .= ',';
                    } 
                    $aclGroupIdsCondition .= ' :acl_' . $aclId;
```

Reply:
seen together, we see later